### PR TITLE
feeds: Rename to gnome-feeds

### PR DESCRIPTION
**Pull request by doronbehar**
Make it reach repology's `gnome-feeds` project:

https://repology.org/project/gnome-feeds

Ref: https://github.com/NixOS/nixpkgs/pull/92339#issuecomment-817111748

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dotlambda** [comment]
Why gnome-feeds if the upstream project is called gfeeds and other distros do not unanimously call this gnome-feeds?

File: pkgs/applications/networking/feedreaders/gnome-feeds/default.nix [hunk 1]
```diff
@@ -17,14 +17,15 @@
...
 python3.pkgs.buildPythonApplication rec {
-  pname = "feeds";
+  pname = "gfeeds";
   version = "0.16.1";
 
   src = fetchFromGitLab {
     domain = "gitlab.gnome.org";
     owner = "World";
-    repo = "gfeeds";
+    repo = pname;
```
Comment (dotlambda) [inline pkgs/applications/networking/feedreaders/gnome-feeds/default.nix:28]: No need to change this. And I think we shouldn't cause the repo could be called anything, its name doesn't change if `pname` does.

**Review by dotlambda — CHANGES_REQUESTED** [review]
We should add an alias for `feeds`.

File: pkgs/applications/networking/feedreaders/gnome-feeds/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/applications/networking/feedreaders/gnome-feeds/default.nix:28]: Also not a big fan of this.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
You are missing an entry into aliases.nix.

**Comment by SuperSandro2000** [comment]
> Why gnome-feeds if the upstream project is called gfeeds and other distros do not unanimously call this gnome-feeds?

Well, Debian does https://sources.debian.org/src/gnome-feeds/0.16.2+dfsg1-1/

**Comment by dotlambda** [comment]
> Well, Debian does https://sources.debian.org/src/gnome-feeds/0.16.2+dfsg1-1/

That doesn't contradict my statement.

**Comment by doronbehar** [comment]
> Why gnome-feeds if the upstream project is called gfeeds and other distros do not unanimously call this gnome-feeds?

We have to decide on one of these options. I took `gnome-feeds` because `gfeeds` is only used in the url of the repo, and the majority of the distros use this name. `gfeeds` seems like a name used only internally by the Gnome foundation. Even the README uses `feeds` as the name of the project.

> You are missing an entry into aliases.nix.

From what I know and read:

https://github.com/NixOS/nixpkgs/blob/a73020b2a150322c9832b50baeb0296ba3b13dd7/pkgs/top-level/aliases.nix#L33-L35

Aliases are meant for backwards compatibility but in this case the package hasn't reached stable yet so I think we don't need an alias.

File: pkgs/applications/networking/feedreaders/gnome-feeds/default.nix (hunk 1 above)
Comment (doronbehar) [reply pkgs/applications/networking/feedreaders/gnome-feeds/default.nix]: > Also not a big fan of this.

Removed.

**Comment by SuperSandro2000** [comment]
pname should match the attribute name.

**Comment by doronbehar** [comment]
> pname should match the attribute name.

Fixed.

**Review by dotlambda — APPROVED** [review]

**Pull request merged**